src/main/java/de/codingair/tradesystem/trade/Trade.java

```diff
@@ -287,9 +295,6 @@ public void run() {
                 guis[0].close();
                 guis[1].close();
 
-                Profile p0 = TradeSystem.getProfile(players[0]);
-                Profile p1 = TradeSystem.getProfile(players[1]);
-
```

Comment:
Why do you remove the Profile variables?

Reply:
I used them at the money check, so I just moved them upwards.

Reply:
Oh yeah, sorry